packages/centreon-ui/src/utils/useCopyToClipboard.ts

```diff
@@ -0,0 +1,33 @@
+import useSnackbar from '../Snackbar/useSnackbar';
+
+type CopyFn = (text: string) => void;
```

Comment:
It's an async function
```suggestion
type CopyFn = (text: string) => Promise<void>;
```